server/src/main/java/io/druid/metadata/storage/derby/DerbyConnector.java

```diff
@@ -41,6 +43,8 @@ public DerbyConnector(Supplier<MetadataStorageConnectorConfig> config, Supplier<
     datasource.setDriverClassName("org.apache.derby.jdbc.ClientDriver");
 
     this.dbi = new DBI(datasource);
+
+    log.info("Configured Derby as the metadata storage");
```

Comment:
minor nitpick, can we remove "the" to be consistent with the deep storage log messages?